pages/app-layout/multi-layout-with-table-sticky-header.page.tsx

```diff
@@ -0,0 +1,58 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+
+import AppLayout from '~components/app-layout';
+import Button from '~components/button';
+import Header from '~components/header';
+import Table from '~components/table';
+
+import { generateItems, Instance } from '../table/generate-data';
+import { columnsConfig } from '../table/shared-configs';
+import ScreenshotArea from '../utils/screenshot-area';
+import { Breadcrumbs, Navigation } from './utils/content-blocks';
+import { Tools } from './utils/content-blocks';
+import labels from './utils/labels';
+import * as toolsContent from './utils/tools-content';
+
+const items = generateItems(100);
+
+export default function () {
+  return (
+    <ScreenshotArea gutters={false}>
```

Comment:
Need to ensure this dev page passes new integration tests added there: https://github.com/cloudscape-design/components/pull/3063

Reply:
It does